packages/esm-patient-programs-app/translations/en.json

```diff
@@ -21,6 +21,7 @@
   "fullyEnrolled": "Enrolled in all programs",
   "location": "Location",
   "noEligibleEnrollments": "There are no more programs left to enroll this patient in",
+  "noProgramsConfigured": "No programs configured",
```

Comment:
Theoretically should be two translation strings - one for the title and one for the subtitle